libs/sdk-ui-dashboard/src/_staging/drills/test/InsightDrillDefinitionUtils.test.ts

```diff
@@ -21,6 +20,7 @@ import {
     ComplexDashboardListed,
     ComplexDashboardWithReferences,
 } from "../../../model/tests/fixtures/ComplexDashboard.fixtures";
+import { newDisplayFormMap, newInsightMap, newMapForObjectWithIdentity } from "../../metadata/objRefMap";
```

Comment:
staging code using mocks from dashboard model.. this is going to kick us three steps ahead

Reply:
Going to delete this test all scenarios are covered as handler test.